lighthouse-core/test/results/sample_v2.json

```diff
@@ -1450,17 +1488,29 @@
               "type": "node",
               "selector": "body > a:nth-child(15)",
               "path": "3,HTML,1,BODY,27,A",
-              "snippet": "<a href=\"javascript:void(0)\" target=\"_blank\">"
+              "snippet": "<a href=\"javascript:void(0)\" target=\"_blank\"></a>",
+              "axeFailureSummary": "Fix all of the following:\n  Element is in tab order and does not have accessible text\n\nFix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible\n  Element's default semantics were not overridden with role=\"presentation\"\n  Element's default semantics were not overridden with role=\"none\""
             }
           },
           {
             "node": {
               "type": "node",
               "selector": "body > a[href$=\"mailto:inbox@email.com\"]",
               "path": "3,HTML,1,BODY,29,A",
-              "snippet": "<a href=\"mailto:inbox@email.com\" target=\"_blank\">"
+              "snippet": "<a href=\"mailto:inbox@email.com\" target=\"_blank\"></a>",
+              "axeFailureSummary": "Fix all of the following:\n  Element is in tab order and does not have accessible text\n\nFix any of the following:\n  Element does not have text that is visible to screen readers\n  aria-label attribute does not exist or is empty\n  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty or not visible\n  Element's default semantics were not overridden with role=\"presentation\"\n  Element's default semantics were not overridden with role=\"none\""
```

Comment:
@joelhsmith it seems like in several cases these can be static

Do you have a sense of which cases they're needed per-node? Is it just color-contrast?

Reply:
Looking through [axe-core](https://github.com/dequelabs/axe-core/tree/020fd150ed0bf0702a89d19712bd38ea0c3817d8/lib/checks), I found these:

aria-allowed-attr
aria-required-children
aria-required-parent
aria-valid-attr-value
aria-valid-attr
color-contrast
fieldset
group-labelledby
duplicate-id
non-empty-if-present

I would not be surprised in the future if even more of them became contextual in the future. Plus people can write their own checks which also might be contextual.